File: src/main/java/com/example/matchteam/dto/board/comment/FindCommentDto.java
package com.example.matchteam.dto.board.comment;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import java.sql.Timestamp;

@Getter
@Setter
@ToString
public class FindCommentDto {
    private String content;
    private String name;
    private Timestamp createdDate;

    @Builder
    public FindCommentDto(String content, String name, Timestamp createdDate) {
        this.content = content;
        this.name = name;
        this.createdDate = createdDate;
    }
}

File: MatchingTeam/app/src/main/java/com/example/matchingteam/dto/board/comment/FindCommentDto.kt
package com.example.matchingteam.dto.board.comment

import com.google.gson.annotations.SerializedName
import java.sql.Timestamp

data class FindCommentDto(
    @SerializedName("content") val content: String,
    @SerializedName("name") val name: String,
    @SerializedName("createdDate") val createdDate: Timestamp
)
